parts/kubernetesmastercustomscript.sh

```diff
@@ -165,6 +167,11 @@ function writeKubeConfig() {
     chmod 700 $KUBECONFIGDIR
     chmod 600 $KUBECONFIGFILE
 
+    FQDNSuffix="cloudapp.azure.com"
+    if [ "$TARGET_ENVIRONMENT" = "AzureChinaCloud" ]
```

Comment:
For example, for any of this to work for Windows, we will need the FQDN Suffix in the windows files as well. This is something that should probably be in the internal model.